modules/ROOT/pages/single-resource-transaction.adoc

```diff
@@ -18,7 +18,7 @@ of the JMS connectors.
 </flow>
 ----
 
-The configuration above defines a JMS message source that receives on a "test.in" queue and another JMS operation that sends on a "test.out" queue. The transactionalAction attribute dictates how Mule initiates a transaction. Within this flow, Mule starts a new transaction for every message it receives (message source), and always joins a transaction in progress for every message it sends (publish operation).  Mule commits only those messages which successfully pass through the complete flow; if at any point in the flow a message throws an error which is propagate (i.e. is not handled by an `on-error-continue`), Mule rolls back the transaction. Refer to xref:on-error-scope-concept.adoc[On Error Scope]for details on how to handle errors. 
+The configuration above defines a JMS message source that receives on a `test.in` queue and another JMS operation that sends on a `test.out` queue. The `transactionalAction` attribute dictates how Mule initiates a transaction. Within this flow, Mule starts a new transaction for every message it receives (message source), and **always joins** the transaction in progress for every message it sends (publish operation). Mule commits only those messages which successfully pass through the complete flow; if at any point in the flow a message throws an error which is propagate (i.e. is not handled by an `on-error-continue`), Mule rolls back the transaction. Refer to xref:on-error-scope-concept.adoc[On Error Scope]for details on how to handle errors. Both JMS source and publish operation must use the same connection configuration, since this mechanism is intended to manage one resource.
```

Comment:
(publish operation) -> (`jms:publish` operation)

Reply:
```suggestion
The configuration above defines a JMS message source that receives on a `test.in` queue and another JMS operation that sends on a `test.out` queue. The `transactionalAction` attribute dictates how Mule initiates a transaction. Within this flow, Mule starts a new transaction for every message it receives (message source), and **always joins** the transaction in progress for every message it sends (`jms:publish` operation). Mule commits only those messages which successfully pass through the complete flow; if at any point in the flow a message throws an error which is propagate (i.e. is not handled by an `on-error-continue`), Mule rolls back the transaction. Refer to xref:on-error-scope-concept.adoc[On Error Scope]for details on how to handle errors. Both JMS source and publish operation must use the same connection configuration, since this mechanism is intended to manage one resource.
```